lib/aggregators/block.ts

```diff
@@ -42,18 +62,25 @@ class BlockAggregator extends Aggregator {
       { concurrency: 1 }
     );
 
-    block.nameOperations = block.nameOperations.filter(Boolean);
+    const block: BlockAggregatorResult = {
+      ...blockchainInfoBlock,
+      transactions: blockchainInfoBlock.transactions.slice(0, 10),
+      nameOperations: nameOperations.filter(Boolean)
+    };
 
     return block;
   }
 
-  static expiry() {
+  expiry() {
     return 60 * 60 * 24 * 2; // 2 days
   }
 
-  static verbose(hash, multi) {
+  verbose(hash: BlockAggregatorOpts, multi?: multi) {
```

Comment:
You can probably just remove this fn, `multi` was from a pre-postgres time when certain jobs took forever, so I setup a progress bar stdout thing.

Reply:
Makes sense -- will do in a follow up PR to avoid merge conflicts with pending PRs